README.md

```diff
@@ -26,38 +26,38 @@ Privacy-focused, free, powerful crypto tax calculator
 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=I%20use%20RP2,%20the%20privacy-focused,%20open%20source,%20free,%20non-commercial%20crypto%20tax%20calculator&url=https://github.com/eprbell/rp2/?anything)
 
 ## Table of Contents
-* **[Introduction](https://github.com/eprbell/rp2/tree/main/README.md#introduction)**
```

Comment:
This won't work on Pypi: when the README is displayed there all these links require the full URL. Same thing in the rest of the file.